_testdata/regression.yml

```diff
@@ -25,59 +25,59 @@
   Statements:
     - SELECT repository_id,contributor_count FROM (SELECT repository_id, COUNT(DISTINCT commit_author_email) AS contributor_count FROM commits GROUP BY repository_id) AS q ORDER BY contributor_count DESC LIMIT 10
 -
-  ID: 'query06'
+  ID: 'query05'
```

Comment:
Delete all queries that scan all `files` table as they take too much time to test.